grpc-native/src/main/java/org/ballerinalang/net/grpc/builder/syntaxtree/SyntaxTreeGenerator.java

```diff
@@ -212,8 +227,23 @@ public static SyntaxTree generateSyntaxTree(StubFile stubFile) {
 
     public static SyntaxTree generateSyntaxTreeForServiceSample(ServiceStub serviceStub, boolean addListener) {
         NodeList<ModuleMemberDeclarationNode> moduleMembers = AbstractNodeFactory.createEmptyNodeList();
+        Collection<ImportDeclarationNode> nodes = new LinkedHashSet<>();
```

Comment:
We can create an empty node list using `AbstractNodeFactory` and keep adding to the list. eg: 
`NodeList<ImportDeclarationNode> imports = AbstractNodeFactory.createEmptyNodeList();`

Reply:
fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/312/commits/2ed157a55d4c98f760401d3e5546e856efe6a738